backend.native/tests/build.gradle

```diff
@@ -287,10 +287,11 @@ def createTestTasks(File testRoot, Class<Task> taskType, Closure taskConfigurati
 
 void dependsOnPlatformLibs(Task t) {
     if (!useCustomDist) {
-        t.dependsOn(':distPlatformLibs')
         def testTarget = project.testTarget
         if (testTarget != null && testTarget != project.hostName) {
             t.dependsOn(":${testTarget}PlatformLibs")
+        } else {
+            t.dependsOn(':distPlatformLibs')
```

Comment:
Can this affect `sanity` task definition?

Reply:
You're right, it may. I'll fix it.